admin/class-metabox.php

```diff
@@ -151,6 +151,20 @@ public function setup_page_analysis() {
 							$this,
 							'column_sort',
 						), 10, 2 );
+
+						/*
+						 * Use the `get_user_option_{$option}` filter to change the output of the get_user_option
+						 * function for the `manage{$screen}columnshidden` option, which is based on the current
+						 * admin screen. The admin screen we want to target is the `edit-{$post_type}` screen.
+						 */
+						$filter = sprintf(
+							'get_user_option_%s',
+							sprintf(
+								'manage%scolumnshidden',
```

Comment:
Is this translatable? If so, add translator comments, example: https://github.com/Yoast/wordpress-seo/pull/2584/files#diff-e9e81ea2561a3b1132aa4f0841038c0cR160